tests/unit/composables/Project.spec.js

```diff
@@ -438,4 +440,11 @@ describe('Test composable: Project', () => {
       expect(GitEvent.PushEvent.next).toBeCalled();
     });
   });
+
+  describe('Test function: gitAdd', () => {
+    it('should call git add', async () => {
+      const result = await gitAdd('projectId', 'filepath');
+      expect(result).toEqual('add');
```

Comment:
...and here you should check if the git.add have been called (bonus: check the params given)

Reply:
done